src/common/selectors.js

```diff
@@ -0,0 +1,10 @@
+import { createSelector } from 'reselect';
```

Comment:
Note this is in the common module - meaning it should ultimately be shared across MFEs.